src/aerie_cli/utils/sessions.py

```diff
@@ -136,6 +137,7 @@ def start_session_from_configuration(
         username (str, optional): Aerie username.
         password (str, optional): Aerie password.
         secret_post_vars (Dict[str, str], optional): Optionally provide values for some or all secret post request variable values. Defaults to None.
+        override (bool, optional): Override Aerie host version check. Defaults to False.
```

Comment:
Nonblocking, feel free to resolve: the name feels a little confusing, as I would expect an override to be setting a value, not ignoring the results of a check. Maybe `force`, as that feels similar to how for other bash CLIs, the `force/f` flag is used to forcibly execute a command that would normally except?

Reply:
Agree, changed to `--force`